src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -79,6 +88,48 @@ export default function LatestProgressUpdateCardPresentation({
                             </span>
                         </span>
                     </Tooltip>
+                    {isDevMode &&
+                        (username === '' ? (
+                            <img
+                                src={userProfileImageUrl}
+                                alt={'Avatar'}
+                                data-testid="latest-progress-update-card-profile-picture"
+                                className={
+                                    styles[
+                                        'latest-progress-update-card__profile-picture'
+                                    ]
+                                }
+                            />
+                        ) : (
+                            <Tooltip
+                                content={username}
+                                tooltipPosition={{
+                                    top: '-2.2rem',
+                                    right: '-3.8rem',
+                                    width: '10rem',
+                                }}
+                            >
+                                <a
+                                    href={`${USER_MANAGEMENT_URL}?username=${username}`}
```

Comment:
I thinkWe can use Link instead of a as we are using nextjs

Reply:
We can but we are redirecting user to different website and I think Link will provide any additional benefits in this case.

Reply:
But since we are using next and next suggest to use Link over <a>

Reply:
We use link in place of a for internal routes but If you strongly prefer <Link>, I will update it

Reply:
If this is external then keep it anchor then